scss/forms/_form-check.scss

```diff
@@ -130,7 +130,12 @@
     margin-left: $form-switch-padding-start * -1;
     background-image: var(--#{$prefix}form-switch-bg);
     background-position: left center;
-    @include border-radius($form-switch-border-radius);
+    @if $enable-rounded {
```

Comment:
I haven't tested it, I'll make a more thorough review later, but the mixin definition is the following:

```scss
@mixin border-radius($radius: $border-radius, $fallback-border-radius: false) {
  @if $enable-rounded {
    border-radius: valid-radius($radius);
  }
  @else if $fallback-border-radius != false {
    border-radius: $fallback-border-radius;
  }
}
```

Just by reading the code, a simple `@include border-radius($form-switch-border-radius, 0);` could maybe work 🤷

Reply:
does the same, only simpler.